scripts/release/rpm/ubi.dockerfile

```diff
@@ -6,8 +6,11 @@ ARG image=registry.access.redhat.com/ubi8/ubi:8.4
 FROM ${image} AS build-env
 ARG cli_version=dev
 ARG python_package=python39
+ARG python_cmd=python3.9
 
 RUN yum update -y
+ARG image
```

Comment:
`image` is out of scope, declare again. https://docs.docker.com/engine/reference/builder/#scope